src/Controls/tests/DeviceTests/Elements/Shell/ShellTests.Windows.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using System;
+using System.Collections;
```

Comment:
Nice, thanks for include the tests. Could we also update the MAUI Gallery to have a way to test it also running the sample?

Reply:
I add `AsMultipleItems` to the second flyout so you can now test this behavior there.

=> Select Tab1
=> Now select one of the children from Tab2
=> check that in the flyout `Tab2` is now selected